src/unsafe/unsafe-traits.md

```diff
@@ -0,0 +1,32 @@
+# Implementing Unsafe Traits
+
+The `zerocopy` crate has an unsafe trait that looks
```

Comment:
You should probably say something like "A trait can be marked as _unsafe_ if it has extra preconditions which the implementer must uphold. Here is an example". You can then explain to the instructor that this comes from zerocopy since this fact doesn't seem central to _what_ an unsafe trait is.

Reply:
Done.